squad/frontend/urls.py

```diff
@@ -36,4 +36,5 @@
     url(r'^(%s)/(%s)/build/([^/]+)/testrun/([^/]+)/metadata$' % ((slug_pattern,) * 2), views.test_run_metadata, name='testrun_metadata'),
     url(r'^(%s)/(%s)/build/([^/]+)/testrun/([^/]+)/attachments/([^/]+)$' % ((slug_pattern,) * 2), views.attachment, name='attachment'),
     url(r'^testjob/(\d+)$', views.test_job, name='test_job'),
+    url(r'^(%s)/(%s)/update-annotation/$' % ((slug_pattern,) * 2), views.update_annotation, name='update_annotation'),
```

Comment:
do we consider annotations 'sensitive'? If not, maybe create REST endpoint and update using PUT requests?